app/models/move.rb

```diff
@@ -89,6 +90,19 @@ def cancel(reason: CANCELLATION_REASON_OTHER, comment: nil)
     )
   end
 
+  def rebook
+    self.class.new(
```

Comment:
I think this should only be `new` if it is not already rebooked - otherwise it should return the existing rebook record

Reply:
Done 👍